zcash_client_backend/src/data_api/chain.rs

```diff
@@ -326,46 +325,10 @@ where
         None
     };
 
-    let mut continuity_check_metadata = prior_block_metadata;
     block_source.with_blocks::<_, DbT::Error>(
         Some(from_height),
         Some(limit),
         |block: CompactBlock| {
-            // check block continuity
```

Comment:
Huh, did we move the continuity check information into the scanner, but forget to remove it from the loader? I guess that checking it here means we avoid some unnecessary work if there's a connection problem, but I think in general it is fine integrated into the scanner.